entity-management-web/src/main/java/eu/europeana/entitymanagement/web/service/impl/EntityRecordService.java

```diff
@@ -343,4 +368,62 @@ else if (fieldValuePrimaryObject != null) {
     public void dropRepository(){
     	this.entityRecordRepository.dropCollection();
 	}
+
+    private void setEntityAggregation(EntityRecord entityRecord, String entityId) {
+        // set isAggregatedBy on Entity
+        Aggregation isAggregatedBy = new AggregationImpl();
+        isAggregatedBy.setId(entityId + "#aggregation");
+        isAggregatedBy.setCreated(new Date());
```

Comment:
I would add a date parameter to the method interface to ensure that all aggregations have the same creation and modified date, when the registerEntity method is called.